nixos/modules/services/continuous-integration/github-runner/service.nix

```diff
@@ -181,9 +181,6 @@ with lib;
       "${stateDir}/${currentConfigTokenFilename}"
     ];
 
-    # By default, use a dynamically allocated user
-    DynamicUser = true;
-
```

Comment:
Why do you want to remove `DynamicUser=`? Cause you think it's redundant anyway?

Reply:
It's re-added conditionally at the bottom